Jenkinsfile.release

```
@@ -581,6 +554,17 @@ void addSkipTestsParam(buildParams){
     addBooleanParam(buildParams, 'SKIP_TESTS', params.SKIP_TESTS)
 }
 
+void addMavenSettingsParam(buildParams){
+    addStringParam(buildParams, 'MAVEN_SETTINGS_CONFIG_FILE_ID', params.MAVEN_SETTINGS_CONFIG_FILE_ID)
+}
+
+void addMavenNexusReleaseParam(buildParams){
```

Maybe use a plural to signal there is more than a single property?
```suggestion
void addMavenNexusReleaseParams(buildParams){
```